script.js

```diff
@@ -1,23 +1,37 @@
+var currentGameMode = "Please key in your name.";
+var userName = "";
+var numberOfWins = 0;
+var numberOfLose = 0;
+
 var main = function (input) {
-  if (input != "Scissors" && input != "Stone" && input != "Paper") {
-    return "Invalid input. Please select either 'Scissors', 'Paper', or 'Stone'.";
-  }
-  var myOutputValue = "The program won!";
-  var gameOutput = randomSPS();
-  console.log(gameOutput);
-  if (input == gameOutput) {
-    myOutputValue = "It is a draw!";
-  }
-  if (
-    (input == "Scissors" && gameOutput == "Paper") ||
-    (input == "Paper" && gameOutput == "Stone") ||
-    (input == "Stone" && gameOutput == "Scissors")
-  ) {
-    myOutputValue = "You won!";
+  var myOutputValue = "";
+  if (currentGameMode == "Please key in your name.") {
+    userName = input;
+    currentGameMode = "dice game";
+    myOutputValue = "Hello " + userName + "!";
```

Comment:
I would recommend always using the template literal over quotations when combining strings with variables like you did later. Even if its just a small statement, its always good to be consistent in your code layout